ContactsApp/Contact.swift

```diff
@@ -2,14 +2,15 @@
 //  Contact.swift
 //  ContactsApp
 //
-//  Created by Davide Mendolia on 30/09/15.
-//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//  Created by Rohit Bansal on 04/10/15.
 //
 
 import Foundation
 
-protocol Contact {
-    var firstName: String { get }
-    var lastName: String { get }
-    var phonenumber: String { get }
+struct Contact : ContactProtocol {
```

Comment:
The difference is type passed through each layer of the application. Where we can isolate the implementation of Contact at each layer if we keep the protocol instead of the concrete type.